sql_it_test.go

```diff
@@ -820,8 +892,8 @@ func testSQLQuery(t *testing.T, keyFmt, valueFmt string, keyFn, valueFn func(i i
 		query := fmt.Sprintf(`SELECT __key, this FROM "%s" ORDER BY __key`, mapName)
 		var result sql.Result
 		sqlService := client.SQL()
-		if stmt.SQL == "" {
-			stmt.SQL = query
+		if stmt.SQL() == "" {
+			it.Must(stmt.SetSQL(query))
```

Comment:
Why is this needed?

Reply:
I think it was designed to execute a default query when no query is provided via statement.